src/theme/FilterTabs.ts

```diff
@@ -8,7 +8,9 @@ export const FilterTabs: ComponentStyleConfig = {
   },
   variants: {
     primary: (props) => ({
-      bgColor: mode("white", "gray.900")(props),
+      bgColor: mode("white", "gray.800")(props),
+      border: "1px solid",
+      borderColor: "gray.100",
```

Comment:
@georgeweiler  Based on Figma the border color of FilterTabs for "primary" vairant should be `E2E8F0` which is `gray.100`. I fixed that in c49a1d281e219cfb33dcf2f7394ba60290b10c09